scripts/automation/Radius/Functions/Private/UserJson/Get-UserJsonData.ps1

```diff
@@ -0,0 +1,31 @@
+function Get-UserJsonData {
+    [OutputType([System.Collections.ArrayList])]
+    [CmdletBinding()]
+    param (
```

Comment:
No parameter here

Reply:
This is an odd case but since I'm explicitly telling powershell to export a type `System.Collections.ArrayList` you do need to specify a param set, even if it's null